core/Metrics/Formatter.php

```diff
@@ -188,6 +188,26 @@ public function getPrettyMoney($value, $idSite)
         return $prettyMoney;
     }
 
+    public function getPrettyBytes($bytes)
+    {
+        $units = array(
+            'B' => 1,
+            'KB' => 1024,
+            'MB' => pow(1024, 2),
+            'GB' => pow(1024, 3),
+            'TB' => pow(1024, 4),
+        );
+
+        $prettyBytes = '';
+        foreach ($units as $unit => $factor) {
+            if ($bytes > $factor) {
+                $prettyBytes = round($bytes / $factor, 2) . $unit;
+            }
+        }
+
+        return $prettyBytes;
+    }
```

Comment:
Cheers, I searched for it but didn't find the existing one. Will use the existing one. Might change the existing one to remove the space before the unit and use 'GB' instead of 'G' just to be clearer